src/components/Accordion/AccordionItem.tsx

```diff
@@ -40,6 +40,10 @@ export interface AccordionItemProps extends React.HTMLAttributes<HTMLDivElement>
    * When true applied the transition keyframe animation on item expand. Default as false.
    */
   hasTransition?: boolean;
+  /**
```

Comment:
I'd rather take a "open" boolean and "onOpenChange" callback prop and use that to determine that the component is controlled